lib/pubsub/topic.js

```diff
@@ -144,11 +147,17 @@ Topic.prototype.publish = function(messages, callback) {
   callback = callback || util.noop;
 
   var body = {
-    topic: this.name,
     messages: messages.map(Topic.formatMessage_)
   };
 
-  this.makeReq_('POST', 'topics/publishBatch', null, body, callback);
+  var path = this.name + ':publish';
```

Comment:
I think this is for conforming to the new internal API, which is more rest style. I'll send you the link in an e-mail.